packages/slice-machine/lib/builders/SliceBuilder/Header/SimulatorButton/index.tsx

```diff
@@ -175,24 +174,24 @@ const SimulatorButton: React.FC<{
       >
         <Button
           data-tip
-          Icon={BsPlayCircle}
-          iconFill={"#4E4E55"}
-          label="Simulate Slice"
           data-testid="simulator-open-button"
           onClick={() => {
             onCloseToolTip();
             window.open(`${router.asPath}/simulator`, SIMULATOR_WINDOW_ID);
           }}
           disabled={disabled}
-          variant={"secondary"}
-          sx={{
-            color: "#4E4E55",
-            backgroundColor: "#F3F5F7",
-            border: "1px solid #3E3E4826",
-            height: 40,
-            ...(disabled ? { opacity: 0.5, cursor: "not-allowed" } : {}),
-          }}
-        />
+          startIcon={
+            <PlayCircleIcon
+              style={{
+                // TODO(DT-1538): our icons should have a `viewBox` of 24px.
```

Comment:
I think most of the icons take SVG props ?

Reply:
you can pass the `viewBox` as a prop and it should override it (although it might look weird since `veiwBox` basically sets the coordination upper and lower limits.

Are you sure it's not the width and height you want to change?